### ISSUE TITLE
Faster select tool keybind (when selecting consecutively)

### ISSUE THREAD

**Issue by wiryfuture** [flxzt/rnote#469]
I have the select tool bound to one of my stylus buttons, which works fine for when i want to select stuff once, but if I want to select something else, I want to be able to press the selection keybind again to forget what i selected earlier and give me the selection tool again. 
Currently, pressing the select tool when you have already selected some content does nothing (technically it goes back to the pen tool, which makes even less sense). The fastest way to select content again  is to press the other stylus button to switch to the eraser tool and then press the select button again.

TLDR; stylus keybind for select tool should start new selection even when you have already selected stuff.

**Comment by flxzt** [comment]
I think this should be fixed / implemented by the "toggle" button mode
